test/unit/foreman/renderer/scope/macros/host_template_test.rb

```diff
@@ -104,6 +105,7 @@ class HostTemplateTest < ActiveSupport::TestCase
       @scope.instance_variable_set('@host', host)
       FactoryBot.create(:parameter, :name => 'false_param', :value => "false")
       assert @scope.host_param_false?('false_param')
+      refute @scope.host_param_true?('true_param')
```

Comment:
This should probably be 
```suggestion
      refute @scope.host_param_false?('true_param')
```